pages/data.js

```diff
@@ -30,9 +32,25 @@ export default class Explore extends React.Component {
     this.updateFilterGroup = this.updateFilterGroup.bind(this);
   }
 
+  static getInitialProps({ query }) {
+    return { query };
+  }
+
+  /**
+   * Once component mounts, check for query string and load
+   * dataset if it exists, else load the first dataset.
+   */
   componentDidMount() {
+    const { query } = this.props;
```

Comment:
Curious: what happened when you tried to access the query params through `next/router`? Error message, nothing?